app/scripts/modules/google/serverGroup/configure/serverGroupCommandBuilder.service.js

```diff
@@ -134,7 +134,7 @@ module.exports = angular.module('spinnaker.gce.serverGroupCommandBuilder.service
     function calculateOverriddenStorageDescription(instanceTypeDetails, command) {
       if (instanceTypeDetails.storage.localSSDSupported) {
         if (command.localSSDCount !== instanceTypeDetails.storage.count) {
-          command.viewState.overriddenStorageDescription = command.localSSDCount > 0 ? command.localSSDCount + '×375' : '0';
```

Comment:
The original line results in an odd entry in the instance types table (just a `0`, rather than `0x375` in the screenshot below. I have 1 SSD disk as a default for n2-standard below).

![defaults](https://cloud.githubusercontent.com/assets/13868700/24161694/8552948e-0e3b-11e7-9404-83c6c40d6edd.png)